### DISCUSSION TITLE
Add-ons after ticket purchase?

### DISCUSSION THREAD

**Discussion by jxs714** [pretix/pretix#4443] [Q&A]
Hey guys!

Hope all is well. Is it possible to allow an order to do an add-on after a ticket purchase? If so, what is the easiest way? When I checked a demo order, there was only an option to edit the contact and invoice information. Unless I am missing something

**Comment by jxs714** [comment]
Hey guys, is this possible or will they have to place a whole new order?

**Reply by raphaelm** [reply]
Yes, you can activate it at Settings > Cancellation > Order changes.

**Reply by jxs714** [reply]
Hi Raphael, thanks for the response. I enabled this option and when checking the order confirmation page and trying to make changes, the only option is to change details of the ticket holder or invoice. The link I visit is the order link, for example: https://domain.com/eventname/order/R0TZ9YXR/y2hym24g6wa4boix/

Does this option only work when you allow customers to create accounts?

Screenshot attached

![Order issue](https://github.com/user-attachments/assets/1a33fc3a-e97d-4ba9-808a-be0c2fbd9118)

**Reply by raphaelm** [reply]
No, customer accounts are not a requirement. One of the other preconditions of the feature must be missing. Are cancellations allowed in the product settings of the main product?

**Reply by jxs714** [reply]
I enabled the "Allow product to be canceled or changed" in the settings of that product and the only thing I can do is cancel it from the user side based on the email link that is sent after order. I can only cancel the order or change the details of the user, but I cant change the product or add any others to the order.
